lucet-concurrency-tests/src/killswitch.rs

```diff
@@ -0,0 +1,806 @@
+//! `KillSwitch` has documentation about the correctness of its various edge cases as part of
+//! Lucet's wider documentation, see this section of the book for more:
+//! https://github.com/bytecodealliance/lucet/blob/master/docs/src/lucet-runtime/killswitch.md#implementation-complexities-when-you-have-a-scheduler-full-of-demons
+//!
+//! This module contains actual test cases exercising these conditions where possible. Phrasing
+//! from the above-mentioned section are re-used below in describing test cases. New tests should
+//! have corresponding cases in the Lucet `KillSwitch` chapter!
+use lucet_runtime::vmctx::Vmctx;
+use lucet_runtime::{
+    // lucet_hostcall, Error,
```

Comment:
Perhaps a bit of leftover cruft?

Reply:
heck. yep, I was trying to pare down which imports are necessary and forgot to clean up :(